policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/max_amount.yaml

```diff
@@ -0,0 +1,23 @@
+description: Arkansas has a maximum amount limit for income-tax credits, based on filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas income-tax credit maximum amount 
+  reference:
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | page 26
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf
+  
+  breakdown:
+    - filing_status
+JOINT:
+  2022-01-01: 300
```

Comment:
need to add 2021 values or a 0 is it was introduced in 2022

Reply:
instead we want a file called 

non_refundable.yaml - in credits 

with 2022-01-01: 
-ar_inflation...